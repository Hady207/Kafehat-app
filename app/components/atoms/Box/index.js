import React from 'react';
import { StyleSheet, View } from 'react-native';

const Box = ({ boxStyle, children }) => {
  return <View style={[styles.box, { ...boxStyle }]}>{children}</View>;
};

export default Box;

const styles = StyleSheet.create({
  box: {
    borderTopLeftRadius: 10,
    borderTopRightRadius: 10,
  },
});
