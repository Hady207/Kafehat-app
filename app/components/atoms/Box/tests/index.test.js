// __tests__/Intro-test.js
import React from 'react';
import renderer from 'react-test-renderer';
import Box from '../index';

test('renders correctly', () => {
  const tree = renderer.create(<Box />).toJSON();
  expect(tree).toMatchSnapshot();
});
