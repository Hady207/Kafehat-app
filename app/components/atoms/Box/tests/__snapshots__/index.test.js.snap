// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "borderTopLeftRadius": 10,
        "borderTopRightRadius": 10,
      },
      Object {},
    ]
  }
/>
`;
